threadpool/server/console/src/main/resources/static/index.html

```diff
@@ -1 +1 @@
823ff8","chunk-078a7535":"53ac87fa","chunk-0fde2e8e":"09d1406a","chunk-149a43cf":"53ac87fa","chunk-19132c4b":"8c823ff8","chunk-1b3cdbc8":"53ac87fa","chunk-296c90bf":"8c823ff8","chunk-2d230a36":"31d6cfe0","chunk-2d230fe7":"31d6cfe0","chunk-2e217faa":"8c823ff8","chunk-37b6768d":"8a6d4294","chunk-38e7e764":"a1da0042","chunk-3a6f2dc9":"53ac87fa","chunk-4413401e":"a28bb621","chunk-4f40863a":"53ac87fa","chunk-5428753b":"53ac87fa","chunk-5911c282":"13a7e89e","chunk-60c39f89":"53ac87fa","chunk-9472305a":"31d6cfe0","chunk-a89383d2":"53ac87fa","chunk-adca2a60":"53ac87fa","chunk-d6c1d344":"53ac87fa","chunk-d9fc0e72":"8c823ff8","chunk-eee8a83e":"2b6c99ee","chunk-ef888edc":"5f8941eb"}[e]+".css",u=h.p+t,r=document.getElementsByTagName("link"),f=0;f<r.length;f++){var d=r[f],o=d.getAttribute("data-href")||d.getAttribute("href");if("stylesheet"===d.rel&&(o===t||o===u))return c()}var k=document.getElementsByTagName("style");for(f=0;f<k.length;f++){d=k[f],o=d.getAttribute("data-href");if(o===t||o===u)return c()}var i=document.createElement("link");i.rel="stylesheet",i.type="text/css",i.onload=c,i.onerror=function(c){var t=c&&c.target&&c.target.src||u,r=new Error("Loading CSS chunk "+e+" failed.\n("+t+")");r.request=t,delete a[e],i.parentNode.removeChild(i),n(r)},i.href=u;var l=document.getElementsByTagName("head")[0];l.appendChild(i)})).then((function(){a[e]=0})));var t=u[e];if(0!==t)if(t)c.push(t[2]);else{var r=new Promise((function(c,n){t=u[e]=[c,n]}));c.push(t[2]=r);var d,o=document.createElement("script");o.charset="utf-8",o.timeout=120,h.nc&&o.setAttribute("nonce",h.nc),o.src=f(e),d=function(c){o.onerror=o.onload=null,clearTimeout(k);var n=u[e];if(0!==n){if(n){var t=c&&("load"===c.type?"missing":c.type),a=c&&c.target&&c.target.src,r=new Error("Loading chunk "+e+" failed.\n("+t+": "+a+")");r.type=t,r.request=a,n[1](r)}u[e]=void 0}};var k=setTimeout((function(){d({type:"timeout",target:o})}),12e4);o.onerror=o.onload=d,document.head.appendChild(o)}return Promise.all(c)},h.m=e,h.c=t,h.d=function(e,c,n){h.o(e,c)||Object.defineProperty(e,c,{enumerable:!0,get:n})},h.r=function(e){"undefined"!==typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},h.t=function(e,c){if(1&c&&(e=h(e)),8&c)return e;if(4&c&&"object"===typeof e&&e&&e.__esModule)return e;var n=Object.create(null);if(h.r(n),Object.defineProperty(n,"default",{enumerable:!0,value:e}),2&c&&"string"!=typeof e)for(var t in e)h.d(n,t,function(c){return e[c]}.bind(null,t));return n},h.n=function(e){var c=e&&e.__esModule?function(){return e["default"]}:function(){return e};return h.d(c,"a",c),c},h.o=function(e,c){return Object.prototype.hasOwnProperty.call(e,c)},h.p="/",h.oe=function(e){throw console.error(e),e};var d=window["webpackJsonp"]=window["webpackJsonp"]||[],o=d.push.bind(d);d.push=c,d=d.slice();for(var k=0;k<d.length;k++)c(d[k]);var i=o;n()})([]);</script><script src=/static/js/app.e0306130.js></script></body></html>
```

Comment:
static I will unify the changes later, and this PR can delete the changes

Reply:
restored